corehq/apps/app_manager/views/modules.py

```diff
@@ -114,12 +114,6 @@
 from corehq.apps.domain.models import Domain
 from corehq.apps.fixtures.fixturegenerators import item_lists_by_app, REPORT_FIXTURE, LOOKUP_TABLE_FIXTURE
 from corehq.apps.fixtures.models import LookupTable
-from corehq.apps.hqmedia.controller import MultimediaHTMLUploadController
```

Comment:
I think you can delete the `MultimediaHTMLUploadController` altogether, including `hqmedia/uploader/preview_html_single.html`.

Reply:
Great! Happy to do so,

https://github.com/dimagi/commcare-hq/pull/36764/commits/a9b71fcf58376364a3110508ab9c213ed6ab5d6e